src/forms/BoundSelectField.tsx

```diff
@@ -67,6 +72,12 @@ export function BoundSelectField<T extends object, V extends Value>(
             field.focus();
             maybeCall(onFocus);
           }}
+          fieldDecoration={
+            fieldDecoration ??
+            (isDirtyIndicator && field.dirty
+              ? () => <div css={Css.wPx(8).hPx(8).br100.bgYellow600.$} {...testId.isDirtyIndicator} />
+              : undefined)
```

Comment:
**request:** Can we have a storybook example for this?

Reply:
Maybe now that you've added `play`